helm_deploy/prometheus-custom-rules-resettlement-passport.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  namespace: hmpps-resettlement-passport-dev
+  labels:
+    role: alert-rules
+  name: hmpps-resettlement-passport-external-api-rules
+spec:
+  groups:
+    - name: application-rules
+      rules:
+        - alert: ExternalAPICritical
+          expr: (sum(rate(http_client_requests_seconds_sum{status="CLIENT_ERROR"}[5m])) / sum(rate(http_client_requests_seconds_sum{status="CLIENT_ERROR"}[5m]))) > 0.05
+          for: 5m
+          labels:
+            severity: critical
```

Comment:
This is used to decide which slack channel to send the alert to. Can be set to `{{ .Values.alertSeverity }}` to go the our custom ones.

Reply:
Done